### ISSUE TITLE
 NConsumer Error: Local: Not implemented

### ISSUE THREAD

**Issue by cxphub** [nodefluent/node-sinek#160]
fist time
anyone can help
```bash
[2020-08-19 15:12:26.362] [WARN] console - consumer ERROR { Error: Local: Not implemented
    at Function.createLibrdkafkaError [as create] (/usr/lib/node_modules/sinek/node_modules/node-rdkafka/lib/error.js:334:10)
    at /usr/lib/node_modules/sinek/node_modules/node-rdkafka/lib/kafka-consumer.js:448:29
  message: 'Local: Not implemented',
  code: -170,
  errno: -170,
  origin: 'kafka' }
```

**Comment by rob3000** [comment]
Hi @cxphub what version are you using? How are you creating the NConsumer?

**Comment by cxphub** [comment]
> Hi @cxphub what version are you using? How are you creating the NConsumer?

 Thank you.
It's a castrated node-sinek package.  It's very stable during this year till I started to use java in my new project.
This issue was found when I tried to let node project communicate to the java project. 
**'gzip' compression method was removed **  so this error happened when consumer work.
The java producer  has been changed compression algorithm as "compression.type=lz4", then the problem solved.
Thanks again.
